app/controllers/users_controller.rb

```diff
@@ -0,0 +1,36 @@
+class UsersController < ApplicationController
+  before_action :user, only: %i[show edit update destroy]
+
+  def index
+    @users = User.all
```

Comment:
what will be if you have 1million users, you will load all them? will be usefull to add pagination here, and show about 40 users per page. Please read about will_paginate gem @chu4  is it OK?

Reply:
or kaminary, ect

Reply:
done)